java/keystore-explorer/Portfile

```diff
@@ -0,0 +1,52 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem                  1.0
+PortGroup                   java 1.0
+PortGroup                   github 1.0
+
+github.setup                kaikramer keystore-explorer 5.4.2 v
+
+categories                  java devel security
+platforms                   darwin
+supported_archs             x86_64
+license                     GPL-3
+maintainers                 {@lhaeger gmx.net:lothar.haeger} openmaintainer
+
+description                 KeyStore Explorer is a free GUI replacement for the Java command-line utilities keytool and jarsigner
+long_description            ${description}\
+                            with the following features:\
+    - Create, load, save and convert between various KeyStore types: JKS, JCEKS, PKCS#12, BKS and UBER\
+    - Change KeyStore and KeyStore entry passwords\
+    - Delete or rename KeyStore entries\
+    - Cut/copy/paste KeyStore entries\
+    - Append certificates to key pair certificate chains\
+    - Generate RSA, ECC and DSA key pairs with self-signed X.509 certificates\
+    - Apply X.509 certificate extensions to generated key pairs and Certificate Signing Requests (CSRs)\
+    - View X.509 Certificate, CRL and CRL entry X.509 V3 extensions\
+    - Import and export keys and certificates in many formats: PKCS#12, PKCS#8, PKCS#7, DER/PEM X.509 \
+      certificate files, Microsoft PVK, SPC, PKI Path, OpenSSL\
+    - Generate, view and sign CSRs in PKCS #10 and SPKAC formats\
+    - Sign JAR files\
+    - Configure a CA Certs KeyStore for use with KeyStore operations\
+
+homepage                    https://keystore-explorer.org/
+
+checksums                   rmd160  32a2ed4ee0be4737646abeec8936733ac86d517e \
+                            sha256  6e8bc6e495644406d8f8c73047ae52c5280a22c134cfd7473ec78f69c41f9eec \
+                            size    7247894
+
+java.version                1.8+
+java.fallback               openjdk11
+
+patchfiles                  patch-disable-updatecheck.diff
+
+use_configure               no
+
+depends_build-append        port:gradle
+build                       {
+                            system "GRADLE_USER_HOME=${worksrcpath} ${prefix}/bin/gradle --no-daemon --console=plain -p ${worksrcpath}/kse --stacktrace clean appbundler"
+                            }
+
+destroot                    {
+                            copy ${worksrcpath}/kse/build/appBundle/KeyStore\ Explorer.app ${destroot}${applications_dir}
+                            }
```

Comment:
Your spacing is odd here. Normally in the case of blocks, just go 4 spaces out and do not add spaces for braces. So like this:

```
build {
    system "GRADLE_USER_HOME...
}

```

Reply:
Also, try setting the environment variables using the appropriate port command instead of putting them into the system command. So for example:

`build-env.append GRADLE_USER_HOME=${worksrcpath}
`

Reply:
Again what learned...

Reply:
...only that building fails with build.env-append and works when prefixing the grade system command - reverting.

Reply:
I don't understand why that did not work. But thanks for trying it.